.github/workflows/pr.yaml

```diff
@@ -79,6 +67,17 @@ jobs:
           do
             sleep 3
           done
+      - name: Deploy/Test APIServer embedded KONK
+        timeout-minutes: 5
+        run: |
+          make deploy-example-apiserver KIND_NAME="chart-testing"
+          POD_NAME=$(kubectl get pod -o name -l app.kubernetes.io/name=konk-service )
+          until kubectl exec -it $POD_NAME -- kubectl get contact
+          do
+            kubectl exec -it $POD_NAME -- kubectl describe apiservice v1alpha1.example.infoblox.com
+            sleep 3
+          done
+          make test-example-apiserver
```

Comment:
Why do we run this target twice? The tests already take a long time.

Reply:
Removed